test/vmPerformanceTestFiller.json

```diff
@@ -0,0 +1,38 @@
+{
+    "ackermann33": {
+        "env" : {
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6",
+            "currentNumber" : "0",
+            "currentGasLimit" : "100000000000",
+            "currentDifficulty" : "256",
+            "currentTimestamp" : "1",
+            "currentCoinbase" : "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba"
+        },
+        "pre" : {
+            "0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
+                "balance" : "1000000000000000000",
+                "nonce" : "0",
+                "//" : "contract PerformanceTester {",
```

Comment:
I think it is good idea to add the PerformanceTester.sol file to the repo. I want to add more functions there. Here we can reference just the function name.